Moli/Core/Screens/Profile/ProfileView.swift

```diff
@@ -209,116 +221,104 @@ struct ProfileView: View {
                         UserThumbnailView(userMinified: profileUserImage(), size: .max)
                     }
                     .overlay(Circle().strokeBorder(Color.primaryDark, lineWidth: 3, antialiased: true))
-                    .offset(x: profileImageSizeXOffset, y: profileImageSizeYOffset)
-                }
+                    .offset(x: 0, y: profileImageSizeYOffset)
+                }.padding(.bottom, 40)
```

Comment:
:art: Antog att du ville få effekten om att ha bilden pixel perfect, och det jag såg var att 40 är inte exakt det man vill ha, så jag testa det här och antar att det är värdet du vill ha istället
![simulator_screenshot_8D37AED1-DB62-4380-8C14-7185587DDCB9](https://user-images.githubusercontent.com/10032344/149933689-cf9c9e02-abf3-400a-934a-f922cd33ee80.png)

![simulator_screenshot_F387C3CF-881E-4156-8F47-EA958F424907](https://user-images.githubusercontent.com/10032344/149933741-c1911525-30f3-4dab-bb8c-c08f14adf025.png)

```suggestion
                }
                .padding(.bottom, profileImageSizeYOffset)
```